### ISSUE TITLE
the bot should rename itself before posting from slack to irc

### ISSUE THREAD

**Issue by lagleki** [lojban/slack-irc-syncbot#3]
e.g. if a Slack2IRC bot is called "kluza" and it wants to send a message from Slack from "gleki" it first must say "/nick glekiSLACK" and then send the normal message without any prefixes.

**Comment by djeik** [comment]
I don't think this is a good idea. This creates a lot of "X is now known as Y" messages. Sure, clients can disable those messages, but by default, practically all clients print those.
